docs/en/FAQ.md

```diff
@@ -0,0 +1,100 @@
+
+No. RocketMQ can run independently.
+
+## Usage
+
+### 1. Where does the newly created Consumer ID start consuming messages?
+
+1. If the topic sends a message within three days, then the consumer start consuming messages from the first message saved in the server.
+2. If the topic sends a message three days ago, the consumer starts to consume messages from the latest message in the server, in other words, starting from the tail of message queue.
+3. If such consumer is rebooted, then it starts to consume messages from the last consumption location.
+
+### 2. How to reconsume message when consumption fails?
+
+1. Cluster consumption patternThe consumer business logic code returns Action.ReconsumerLater, NULL, or throws an exception, if a message failed to be consumed, it will retry for up to 16 times, after that, the message would be descarded.
+2. Broadcast consumption patternThe broadcaset consumption still ensures that a message is consumered at least once, but no resend option is provided.
+
+### 3. How to query the failed message if there is a consumption failure?
+
+1. Using topic query by time, you can query messages within a period of time.
+2. Using Topic and Message Id to accurately query the message.
+3. Using Topic and Message Key accurately query a class of messages with the same Message Key.
+
+### 4. Are messages delivered exactly once?
+
+RocketMQ ensures that all messages are delivered at least once. In most cases, the messages are not repeated.
+
+### 5. How to add a new broker?
+
+1. Start up a new broker and register it to the same list of name servers.
+2. By default, only internal system topics and consumer groups are created automatically. If you would like to have your business topic and consumer groups on the new node, please replicate them from the existing broker. Admin tool and command lines are provided to handle this.
+
+## Configuration related
+
+The following answers are all default values and can be modified by configuration.
+
+### 1. How long are the messages saved on the server?
+
+Stored messages are will be saved for up to 3 days, and messages that are not consumed for more than 3 days will be deleted.
+
+### 2. What is the size limit for message Body?
+
+Generally 256KB.
+
+### 3. How to set the number of consumer threads?
+
+When you start Consumer, set a ConsumeThreadNums property, example is as follows:
+```
+consumer.setConsumeThreadMin(20);
+consumer.setConsumeThreadMax(20);
+```
+
+## Errors
+
+### 1. If you start a producer or consumer failed and the error message is producer group or consumer repeat?
+
+Reason：Using the same Producer /Consumer Group to launch multiple instances of Producer/Consumer in the same JVM may cause the client fail to start.
+
+Solution: Make sure that a JVM corresponding to one Producer /Consumer Group starts only with one Producer/Consumer instance.
+
+### 2. If consumer failed to start loading json file in broadcast mode?
```

Comment:
It is better to remove "If" and change the question mark to a full stop.

Reply:
fixed